bp_me/src/include/v/bp_me_cce_mem_if.vh

```diff
@@ -113,7 +114,7 @@ typedef enum bit [2:0]
 
 // CCE-MEM Interface
 `define bp_cce_mem_cmd_payload_width(num_lce_mp, lce_assoc_mp) \
-  (`BSG_SAFE_CLOG2(num_lce_mp)+`BSG_SAFE_CLOG2(lce_assoc_mp))
+  (`BSG_SAFE_CLOG2(num_lce_mp)+`BSG_SAFE_CLOG2(lce_assoc_mp)+`bp_coh_bits)
```

Comment:
bikeshed, but I think $bits(bp_coh_states_e) everywhere is better.  Up to you, though

Reply:
I will fix in a separate change.